### PR TITLE
Update old references to _output/local/go/bin

### PR DESCRIPTION
**Pull request by rhcarvalho**

### CODE REVIEW

**Comment by rhcarvalho** [comment]
@smarterclayton @bparees @mfojtik PTAL

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ ENV PATH $PATH:$GOROOT/bin:$GOPATH/bin
 
 RUN go get github.com/openshift/origin && \
     hack/build-go.sh && \
-    cp _output/local/go/bin/* /usr/bin/ && \
+    cp _output/local/bin/linux/amd64/* /usr/bin/ && \
```
Comment (rhcarvalho) [inline Dockerfile]: I believe we cannot successfully build a release without this change?

Comment (smarterclayton) [inline Dockerfile]: We don't use this Dockerfile to build releases

**Comment by smarterclayton** [comment]
LGTM [merge] thanks

File: Dockerfile (hunk 1 above)
Comment (liggitt) [inline Dockerfile]: Assumes current platform is linux/amd64... Is that valid?

Comment (smarterclayton) [inline Dockerfile]: For Linux today yes, in the future there may be arm architectures.

On Sep 27, 2015, at 11:55 AM, Jordan Liggitt <notifications@github.com>
wrote:

In Dockerfile
<https://github.com/openshift/origin/pull/4816#discussion_r40505922>:

> @@ -17,7 +17,7 @@ ENV PATH $PATH:$GOROOT/bin:$GOPATH/bin
>
>  RUN go get github.com/openshift/origin && \
>      hack/build-go.sh && \
> -    cp _output/local/go/bin/* /usr/bin/ && \
> +    cp _output/local/bin/linux/amd64/* /usr/bin/ && \

Assumes current platform is linux/amd64... Is that valid?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4816/files#r40505922>.

**Pull request merged**